### PR TITLE
ANALYTICS-3090 add_wpc_to_char_summary_api

### PR DESCRIPTION
**Pull request by martin-hristov-acc**

### CODE REVIEW

File: source/includes/_chat_summary.md [hunk 1]
```diff
@@ -357,6 +520,7 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
 |campaign_start_date|String|Start date of campaign|
 |campaign_end_date|String|End date of campaign|
 |intervals|Object|Data for specified interval. [Intervals Object](#chatintervals)|
+<internal>|web_publisher_campaigns|Array|array of web_publisher_campaign objects [Web Publisher Campaign Object](#webpublishercampaign)|</internal>
```
Comment (travisbauer) [inline source/includes/_chat_summary.md:523]: Move the tag inside the table row: `|<internal>web_publisher_campaigns|Array|array of web_publisher_campaign objects [Web Publisher Campaign Object](#webpublishercampaign)</internal>|`

File: source/includes/_chat_summary.md [hunk 2]
```diff
@@ -378,3 +542,20 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
 |chat_service|Integer|Number of chats marked as 'service'|
 |chat_other|Integer|Number of chats marked as 'other'|
 |call_connect|Integer|Number of calls connected|
+
+<internal>
```
Comment (travisbauer) [inline source/includes/_chat_summary.md:546]: Unfortunately we have to tag each element that get's rendered into html.  So tag, `<internal>**Web Publisher Campaign Object**</internal>` and tag the table with `|<internal>Field Name|Datatype|Description</internal>|`

**Review by travisbauer — CHANGES_REQUESTED** [review]

File: source/includes/_chat_summary.md (hunk 1 above)
Comment (martin-hristov-acc) [reply source/includes/_chat_summary.md]: Fixed

File: source/includes/_chat_summary.md (hunk 2 above)
Comment (martin-hristov-acc) [reply source/includes/_chat_summary.md:546]: Fixed

File: source/includes/_chat_summary.md [hunk 3]
```diff
@@ -324,6 +326,167 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
 }
 ```
 
+<internal>
+# Example Response
```
Comment (travisbauer) [inline source/includes/_chat_summary.md:330]: make this `> <internal>Example Response</internal>`

File: source/includes/_chat_summary.md [hunk 4]
```diff
@@ -324,6 +326,167 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
 }
 ```
 
+<internal>
```
Comment (travisbauer) [inline source/includes/_chat_summary.md:329]: remove this

File: source/includes/_chat_summary.md [hunk 5]
```diff
@@ -324,6 +326,167 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
...
+# Example Response
+```javascript
```
Comment (travisbauer) [inline source/includes/_chat_summary.md:331]: make javascript json, and then add <internal> under the json code label.

File: source/includes/_chat_summary.md [hunk 6]
```diff
@@ -324,6 +326,167 @@ https://api.reachlocalservices.com/client_reports/chat_summary/TEST_1?start_date
...
+                    "chat_other": 0,
+                    "call_connect": 0
+                }
+            ]
+        }
+    },
+    "global_master_advertiser_id": "USA_172716"
+}
+```
+</internal>
```
Comment (travisbauer) [inline source/includes/_chat_summary.md:487]: move this to above the triple ticks

**Review by travisbauer — CHANGES_REQUESTED** [review]

**Review by travisbauer — APPROVED** [review]

**Pull request merged**